src/lib/components/explorer/results/ResultsPanel.svelte

```diff
@@ -268,6 +195,16 @@
             active={page.url.pathname.includes('explorer/variant')}
           />
         {/if}
+        {#if showCohortDetails}
```

Comment:
This is more of a question of @emhug 
Should this be first in the list?

Reply:
Thanks for translating @srpiatt! 
Yes, I think if it is straightforward to make the change, Cohort Details should be the first tool shown in the Tool Suite.